functions/index.js

```diff
@@ -8,3 +8,4 @@ if (!function_name || function_name === "verifyRecaptcha") {
 if (!function_name || function_name === "announcements") {
   exports.announcements = require("./controllers/announcements-api/index").service;
 }
+exports.writeToAnalytics = require("./utils/analytics").service;
```

Comment:
Last change, but can we make sure this is similar to how everything else in this file is written to have that conditional check the `function_name === "writeToAnalytics"